pkg/operation/cloudbotanist/alicloudbotanist/controlplane_test.go

```diff
@@ -0,0 +1,50 @@
+package alicloudbotanist
+
+import (
+	"testing"
+
+	corev1 "k8s.io/api/core/v1"
+
+	"github.com/gardener/gardener/pkg/operation"
+	"github.com/gardener/gardener/pkg/operation/common"
+	"github.com/gardener/gardener/pkg/operation/shoot"
+)
+
+func TestRefreshCloudProviderConfig(t *testing.T) {
```

Comment:
Thank's for the test, but please use the ginkgo/gomega framework that we use for all other tests here as well.

Reply:
Done